Cloudfront media purge (#157)
humanmade/altis-cloud: docs/cdn-purge.md

> # CDN Purge

Reviewer: ```suggestion
# Purging Cache on the CDN
```

> The majority of the pages, media viewed on Altis will be cached on CDN to improve performance and delivery time to the user. You can add your own CDN purge rule directly in PHP using `Altis\Cloud\purge_cdn_path` helper function.   

Reviewer: ```suggestion
The majority of the requests, data served by Altis will be cached on the CDN to improve performance and delivery time to the user. In some situations you may need to invalidate specific URLs on the CDN. You can add your own CDN purge rule directly in PHP using `Altis\Cloud\purge_cdn_path` helper function.   
```

> The majority of the requests, data served by Altis will be cached on the CDN to improve performance and delivery time to the user. In some situations you may need to invalidate specific URLs on the CDN. You can add your own CDN purge rule directly in PHP using `Altis\Cloud\purge_cdn_paths` helper function.

Reviewer 2: ```suggestion
The majority of the requests, data served by Altis will be cached on the CDN to improve performance and delivery time to the user. In some situations you may need to invalidate specific URLs on the CDN. You can add your own CDN purge rule directly in PHP using `Altis\Cloud\purge_cdn_paths()` helper function.
```

It's good practice to add a code example of this too so it's clear how to use that function.

## Automatic media purge rule

> By default, Altis will not remove media from CDN on delete attachment. You can enable it from the project `composer.json`. The example below shows the default configuration:

Reviewer: ```suggestion
By default, Altis will not remove uploaded media from the CDN cache when deleting attachments. You can enable this beahviour from the project `composer.json`. The example below shows the default configuration:
```

> By default, Altis will not remove uploaded media from the CDN cache when deleting attachments. You can enable this beahviour from the project `composer.json`. The example below shows the default configuration:

Reviewer 2: ```suggestion
By default, Altis will not remove uploaded media from the CDN cache when deleting attachments. You can enable this behavior from the project's configuration in `composer.json`. The example below shows the default configuration:
```
We're using american english spellings for docs.

```json
{
   "extra": {
      "altis": {
         "modules": {
            "cloud": {
               "cdn-media-purge": false
            }
         }
      }
   }
}
```

> Set `"cdn-media-purge": true` to initiate cdn purge for media on delete attachment.

Reviewer: ```suggestion
Set `"cdn-media-purge": true` to enabled cdn purging of media on when attachments are deleted.
```

> Set `"cdn-media-purge": true` to enabled cdn purging of media on when attachments are deleted

Reviewer 2: ```suggestion
Set `"cdn-media-purge": true` to enable purging media files from the CDN when attachments are deleted.
```
